### PR TITLE
fix tests that involve dates fail when OS culture has 24 hours format

### PR DESCRIPTION
**Pull request by mnieto**
# PR Summary

Inside the vsteam-lib.Test project, tests that have Asserts based on date format (i.e. `actual.CreatedOn.ToString("M/d/yyyy h:mm:ss tt").`) fails when ran in a Windows machine configured with 24h format, for example Spanish. 
See: https://stackoverflow.com/q/38976486/777551

Example of error:
image

The solution is to use an invariant culture so tests will be independent of the local configuration

## PR Checklist

- [x] [Write Help](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#write-help)
- [x] [Write Unit Test](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#write-unit-test)
- [x] [Update CHANGELOG.md](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#update-changelogmd)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ Merged [Pull Request](https://github.com/MethodsAndPractices/vsteam/pull/502) fr
 - Updated test c# project to .NET 6.0
 - Fixed test fail on ubuntu-latest
 
+Merged [Pull Request](https://github.com/MethodsAndPractices/vsteam/pull/506) from [Miguel Nieto](https://github.com/mnieto) the following:
+- fix tests that involve dates fail when OS culture has 24 hours format
+
```
Comment (SebastianSchuetze) [inline CHANGELOG.md]: Can you update to the latest (not released) version 7.12.0?

**Review by SebastianSchuetze — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mnieto) [reply CHANGELOG.md]: updated

**Comment by mnieto** [comment]
@SebastianSchuetze  can you please review and merge

**Comment by mnieto** [comment]
@SebastianSchuetze, can you please review last changes and merge?

**Review by SebastianSchuetze — APPROVED** [review]

**Pull request merged**